cg/meta/workflow/mip.py

```diff
@@ -82,33 +91,16 @@ def check(self, family_obj: models.Family):
                 self.log.warning("%s: %s", flowcell_obj.name, flowcell_obj.status)
         return all(status == "ondisk" for status in statuses)
 
-    def run(self, family_obj: models.Family, **kwargs):
-        """Start the analysis."""
-        if kwargs.get("priority") is None:
-            if family_obj.priority == 0:
-                kwargs["priority"] = "low"
-            elif family_obj.priority > 1:
-                kwargs["priority"] = "high"
-            else:
-                kwargs["priority"] = "normal"
-
-        # skip MIP evaluation of QC criteria if any sample is downsampled/external
-        for link_obj in family_obj.links:
```

Comment:
Where did this one go?

Reply:
@henrikstranneheim  This one isnt being used currently when the execution changed to MIP. WOuld you like to include it in run?